proposal.md

```diff
@@ -389,6 +386,11 @@ containing various builds of R for debugging memory problems. These include
 images with the debugging tools Valgrind or gdb, as well as images with R 
 compiled for use with ASAN, UBSAN or [gctorture](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Using-gctorture-1).
 
+The [CRAN Cookbook](https://contributor.r-project.org/cran-cookbook/) is a 
+guide written in collaboration with the CRAN Team that provides "recipes" for 
```

Comment:
The CRAN team is usually referenced like a single team while their own webpage groups them by task too. Maybe this could reference the "CRAN Submissions team" to be more specific about the kind of content/guidance that can be expected. 

```suggestion
guide written in collaboration with the CRAN Submissions Team that provides "recipes" for 
```